StringCalculatorKata/StringCalculator.cs

```diff
@@ -0,0 +1,35 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace StringCalculatorKata
+{
+    public class StringCalculator
+    {
+        private int Parser(string s)
+        {
+            return int.Parse(s);
+        }
+
+        public int Add(string NumbersString)
+        {
+            if(string.IsNullOrEmpty(NumbersString) || string.IsNullOrWhiteSpace(NumbersString))
+            {
+                return 0;
+            }
+
+            string[] Numbers = NumbersString.Split(",");
```

Comment:
if I wanted Numbers to be a list of integers instead, how would I do that using one LINQ statement?